Makefile

```diff
@@ -1918,15 +1918,15 @@ TEST_APPS=\
 test_apps: distrib
 	@for APP in $(TEST_APPS); do \
 		echo Testing app $${APP}... ; \
-		make -C $(ROOT_DIR)/apps/$${APP} test \
+		$(MAKE) -C $(ROOT_DIR)/apps/$${APP} test \
```

Comment:
ditto, because they report runtimes

Reply:
...reporting runtimes seems more like a job for benchmarks, not tests